rewrite/src/exe/coll.c

```diff
@@ -37,12 +37,6 @@ void COLL_ProjectPointToEdge(SVec3* out, const SVec3* v1, const SVec3* v2, const
     out->y = coords.y;
     out->z = coords.z;
     TEST_COLL_ProjectPointToEdge(v1, v2, point, out);
-    /* This is a hand written assembly function that breaks the ABI,
-    and some callers expect the argument registers to be untouched */
-    __asm__ volatile("move $a0, %0" : : "r"((u32)out));
```

Comment:
Did you ever fully solve this problem? Was it ultimately because of psn00bsdk?

Reply:
Yep, t1 being trashed during the gte registers was causing this bug